### PR TITLE
Add priority in JSONRPC

### PR DESCRIPTION
**Pull request by KaustubhShamshery**
* Add the var priority (by default false)
* Change logic for seed node forwarding if the flag is true

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [x] small-scale cloud test (40 nodes tested on #1302)

### CODE REVIEW

File: src/libMessage/ZilliqaMessage.proto [hunk 1]
```diff
@@ -755,6 +756,13 @@ message LookupSetMicroBlockFromLookup
     required ByteArray signature         = 3;
 }
 
+message LookupForwardTxnsFromSeed
+{
+    optional ProtoTransactionArray shardtransactions = 1;
+    optional ProtoTransactionArray dstransactions    = 2;
+}
```
Comment (ShengguangXiao) [inline src/libMessage/ZilliqaMessage.proto]: Message need to add pubkey and signature to verify lookup authenticity

File: src/libServer/Server.cpp [hunk 2]
```diff
@@ -193,69 +212,85 @@ Json::Value Server::CreateTransaction(const Json::Value& _json) {
...
+          throw JsonRpcException(RPC_INVALID_ADDRESS_OR_KEY,
+                                 "Code is empty and To addr is null");
+          break;
+        default:
+          throw JsonRpcException(RPC_MISC_ERROR, "Txn type unexpected");
       }
     } else {
-      LOG_GENERAL(INFO, "No shards yet");
-      throw JsonRpcException(RPC_IN_WARMUP, "Could not create Transaction");
+      throw JsonRpcException(RPC_IN_WARMUP, "No Shards yet");
```
Comment (ShengguangXiao) [inline src/libServer/Server.cpp]: Where is this exception captured?

File: src/libMessage/ZilliqaMessage.proto (hunk 1 above)
Comment (KaustubhShamshery) [reply src/libMessage/ZilliqaMessage.proto]: message is from seed not lookup. https://github.com/Zilliqa/Issues/issues/425

File: src/libServer/Server.cpp (hunk 2 above)
Comment (KaustubhShamshery) [reply src/libServer/Server.cpp]: captured by default by JsonRpcCpp

**Comment by iantanwx** [comment]
This has been tested and works as expected.

File: src/libMessage/ZilliqaMessage.proto (hunk 1 above)
Comment (ShengguangXiao) [reply src/libMessage/ZilliqaMessage.proto]: Will this issue be fixed in this PR or seperately?

**Pull request merged**